tools/pv_tree_generator/util.py

```diff
@@ -51,4 +141,52 @@ def _read_pop_obs_spec():
         PopObsSpec(pos.pop_type, pos.mprop, pos.stat_type, list(pos.cprop),
                    cpv, pos.name))
   return result
+  
+def _read_stat_var():
+    """Read all the statistical variables"""
+    #read the dcid of all statistical variables
+
+    #currently we read it from the downloaded .csv file
+    #in the future, we will change to query it from bigquery,
+    #or from calling API of datacommons
+    sv_dcid = open("statistical_variables_dcid.txt",'r').read().splitlines()
+
+    #get the triples for each statistical variable and store it as StatVar
+    """
+    example of triples for one statsitical variable
+    ('dc/014es05x0d5l', 'measurementMethod', 'CensusACS5yrSurvey')
+    ('dc/014es05x0d5l', 'constraintProperties', 'income')
+    ('dc/014es05x0d5l', 'income', 'USDollar75000Onwards')
+    ('dc/014es05x0d5l', 'age', 'Years15Onwards')
+    ('dc/014es05x0d5l', 'statType', 'measuredValue')
+    ('dc/014es05x0d5l', 'placeOfBirth', 'BornInStateOfResidence')
+    ('dc/014es05x0d5l', 'measuredProperty', 'count')
+    ('dc/014es05x0d5l', 'incomeStatus', 'WithIncome')
+    ('dc/014es05x0d5l', 'constraintProperties', 'placeOfBirth')
+    ('dc/014es05x0d5l', 'typeOf', 'StatisticalVariable')
+    ('dc/014es05x0d5l', 'populationType', 'Person')
+    ('dc/014es05x0d5l', 'provenance', 'dc/cweckx1')
+    ('dc/014es05x0d5l', 'constraintProperties', 'incomeStatus')
+    ('dc/014es05x0d5l', 'constraintProperties', 'age')
+    """
+    dc.set_api_key(API_KEY)
+    sv_triples = dc.get_triples(sv_dcid)
+    statVars = collections.defaultdict(list)
+    for dcid, triples in sv_triples.items():
+      constraintProperties = set()
```

Comment:
use constraint_properties and check other places for style consistency

Reply:
done